aws/ae-env.yml

```diff
@@ -0,0 +1,92 @@
+Parameters:
+    keyname:
+        Type: String
+        Description: SSH Key Name
+    instancetype:
+        Type: String
+        Description: Instace size
+    amiid:
+        Type: String
+        Description: AMI id
+    envname:
+        Type: String
+        Description: Environment name
+    apiport:
+        Type: Number
+        Description: Port number of epoch node API
+        Default: 3013
+    nodecount:
+        Type: Number
+        Description: Number of epoch nodes
+        Default: 2
+
+Resources:
+
+    ManagementSecurityGroup:
+      Type: 'AWS::EC2::SecurityGroup'
+      Properties:
+          GroupName:
+              Fn::Sub:
+                - "ae-${envname}-management"
+                - { envname: !Ref envname}
+          GroupDescription: Allow node management from internet (ICMP, SSH, API)
+          SecurityGroupIngress:
+          - FromPort: !Ref apiport
+            ToPort: !Ref apiport
+            CidrIp: 0.0.0.0/0
+            IpProtocol: "tcp"
+          - FromPort: 22
+            ToPort: 22
+            CidrIp: 0.0.0.0/0
+            IpProtocol: "tcp"
+
+    EpochSecurityGroup:
+      Type: 'AWS::EC2::SecurityGroup'
+      Properties:
+        GroupName:
+            Fn::Sub:
+              - "ae-${envname}-nodes"
+              - { envname: !Ref envname}
+        GroupDescription: Full access between epoch nodes in their environment
+
+    EpochSecurityGroupIngress:
+      Type: 'AWS::EC2::SecurityGroupIngress'
+      Properties:
+        GroupName: !Ref EpochSecurityGroup
+        FromPort: 0
+        ToPort: 65535
+        SourceSecurityGroupName: !Ref EpochSecurityGroup
+        IpProtocol: "tcp"
+
+    EpochServerLaunchConfiguration:
```

Comment:
The same as the auto scaling group names below, let's have better names related to the environment name

Reply:
it do not have option to set namehttps://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-launchconfig.html